automation/src/test/java/DemoQA/Tests/SliderTests.java

```diff
@@ -0,0 +1,47 @@
+package DemoQA.Tests;
+
+import org.testng.annotations.Test;
+
+import DemoQA.Pages.SliderPage;
+import framework.TestBase;
+
+import static org.testng.Assert.assertTrue;
+
+public class SliderTests extends TestBase {
+  @Test
+  public void canSetTo80() {
+	  SliderPage page = new SliderPage(super.getDriver());
+	  page.navigate();
+	  
+	  page.set(80);
```

Comment:
Move '80' to a variable

https://en.wikipedia.org/wiki/Magic_number_(programming)